src/meltano/core/project_plugins_service.py

```diff
@@ -82,10 +77,7 @@ def __init__(
         instruction = None
 
         if DefinitionSource.HUB in source:
-            instruction = (
-                f"Run `meltano discover {plugin.type.descriptor}` to explore available "
-                "plugins"
-            )
+            instruction = "Check Meltano Hub for available plugins"
```

Comment:
nit

```suggestion
            instruction = "Check https://hub.meltano.com/ for available plugins"
```

Reply:
I thought of doing that, but decided to use `Meltano Hub` instead because they might have their own Hub URL. Then again, if they are using their own Hub URL, they probably know well enough to check that instead of the Hub we host.